tests/recipes/test_full_finetune_distributed.py

```diff
@@ -29,7 +29,10 @@
 
 
 class TestFullFinetuneDistributedRecipe:
-    def _get_test_config_overrides(self):
+    def _get_test_config_overrides(self, optim_in_bwd):
```

Comment:
More of a nit, but I would keep `_get_test_config_overrides` argless. Better to just have something like

```
if optim_in_bwd: 
	cmd.append(["optimizer_in_bwd=True"])
else:
	cmd.append(["clip_grad_norm=100"])
```
e.g. just after L104 below.